index.bs

```diff
@@ -11456,7 +11456,7 @@ used to configure various channel configurations.
             sequence and return.
 
     1. If {{AudioWorkletNodeOptions/outputChannelCount}}
-         [=map/exists=],
+         does not exist,
```

Comment:
Thanks for catching this!

`does not [=map/exists=],` would be the right fix. Could you kindly update the PR?

Reply:
Thanks for the tip. I've just updated the PR.